tests/integration/test_pagure.py

```diff
@@ -146,6 +146,14 @@ def test_pr_info(self):
         assert pr_info.title.startswith("Update Python 2 dependency")
         assert pr_info.status == PRStatus.merged
 
+    def test_update_pr_info(self):
+        self.abiword_project.update_pr_info(
```

Comment:
We have not rights to edit PRs in this repo. We can use some of our projects for that. e.g. [packit](https://src.fedoraproject.org/rpms/packit/) or [ogr](https://src.fedoraproject.org/rpms/python-ogr/).

1. Fork the project.
2. Create some PR by hand. (Or via OGR..;-)
3. Run the tests with the token. -> It should generate you the yaml file in `./tests/integration/tests_data/`
4. Commit the yaml files.

Feel free to ask if there is anything not clear.

If you are not sure about this, I can generate those files for you when you are done with the other code..